Jenkinsfile-pr

```diff
@@ -9,7 +9,7 @@ node('master') {
         echo "Github PR comment: ${env.ghprbCommentBody}"
         if (env.ghprbCommentBody.contains('crc') || env.ghprbCommentBody.contains('ocp4')) {
             NODE_LABEL = "strimzi-crc"
-            DEFAULT_EXCLUDED_GROUPS = ""
+            DEFAULT_EXCLUDED_GROUPS = "loadbalancer"
```

Comment:
Is it intended to disable loadbalancers also for `ocp4`? Wasn't that some AWS infra which supports load balancers?

Reply:
Even if you specify the `ocp4` for the PR, it will deploy the `crc`, so I think this is correct